tests/index.test.js

```diff
@@ -0,0 +1,17 @@
+const { configs, rules } = require('../index');
+const { checkExists } = require('../utils');
```

Comment:
can we use es6 imports here?

Reply:
I tried - doesn't work :(

Reply:
I didn't want to spend to much time on the tooling for now.

Reply:
okok